src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -73,22 +73,18 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownProps, DropdownVariables>
     background: listBackgroundColor,
   }),
 
-  emptyListItem: ({ variables: { listItemBackgroundColor } }) => ({
-    backgroundColor: listItemBackgroundColor,
-  }),
-
-  toggleButton: ({ variables: { toggleButtonSize, width }, props: { fluid } }): ICSSInJSStyle => ({
+  toggleIndicator: ({ props: p, variables: v }): ICSSInJSStyle => ({
     position: 'absolute',
-    height: toggleButtonSize,
-    width: toggleButtonSize,
+    height: v.toggleButtonSize,
+    width: v.toggleButtonSize,
     cursor: 'pointer',
     backgroundColor: 'transparent',
     margin: 0,
     display: 'flex',
     justifyContent: 'center',
     alignItems: 'center',
     userSelect: 'none',
-    ...(fluid ? { right: 0 } : { left: `calc(${width} - ${toggleButtonSize})` }),
+    ...(p.fluid ? { right: 0 } : { left: `calc(${v.width} - ${v.toggleButtonSize})` }),
```

Comment:
Let's rename the variable toggleButtonSize -> toggleIndicatorSize (should we consider whether this will be icon and change the value?!?!)

Reply:
Renamed to `toggleIndicatorSize`